### ISSUE TITLE
Use image alt text as header text as fallback

### ISSUE THREAD

**Issue by yusf** [codimd/server#174]
I have the following scenario:
```
# ![Header](imgsrc)

## Header

### Header
```

Instead of text, I want an image at the top of my document. This screws up the ToC but it doesn't have to if the image comes with an alt text. 

I propose that the app falls back to using the alt text of an image if available.
